### PR TITLE
Remove references to dynamic nat-pool and destination-nat pools from …

### PR DESCRIPTION
**Pull request by agrawalkaushik**
…documentation

- Those concepts were in the functional spec but only the static source-nat-pool was ever
implemented. The documentation has been corrected to reflect that.

### CODE REVIEW

**Review by MichaelBaj — DISMISSED** [review]

File: docs/config_nat.md [hunk 1]
```diff
@@ -64,31 +64,14 @@ Static desination network address translation can be performed by configuring a
...
-  * Supporting more than 49,150 source NAT sessions
-  * Translating LAN traffic to a single IP on the WAN interface, different from the network-interface address
-
-3. Static Destination NAT bindings
-
-  Destination NAT changes the destination address of packets passing through the router. It also offers the option to perform the port translation in transport headers. Destination NAT is mainly used to redirect incoming packets with an external address or port destination to an internal IP address or port inside the network.
-
 ### Static NAT bindings
-A static NAT binding can be configured by creating a `authority > router > nat-pool` object and assigning it to a network-interface.  Be sure to set the pool-type to `static`. The following rules and constraints will apply to this configuration:
+A static NAT binding can be configured by creating a `authority > router > nat-pool` object and assigning it to a network-interface.  The following rules and constraints will apply to this configuration:
```
Comment (Chr1st0ph3rTurn3r) [inline docs/config_nat.md:68]: ...creating an 'authority>router..."

File: docs/config_nat.md [hunk 2]
```diff
@@ -111,61 +94,28 @@ network-interface
...
+In some scenarios, it is desirable to share the same NAT pool across services and interfaces. This feature accommodates the same NAT pool to be configured on different interfaces. There are some considerations that should be noted for failure and recovery.
 
 <u>Flow Move Considerations</u>
 
 Currently session migration is only supported for SVR sessions. This restriction exists because session migration of a non-SVR session is not guaranteed to terminate on the same server. As a result, the remote server might receive mid flow packets from a different source, resulting in undefined behavior.
 
 <u>Session Recovery Considerations</u>
 
-Shared NAT pools provisioned on a HA interface that encur a failover, the 128T software puts the interface in recovery mode to recover all sessions. At the end of the recovery period, all non-discovered ports as designed as free and are returned to the NAT pool.
+Shared NAT pools provisioned on a HA interface that encounters a failover, the 128T software puts the interface in recovery mode to recover all sessions. At the end of the recovery period, all non-discovered ports as designed as free and are returned to the NAT pool.
```
Comment (Chr1st0ph3rTurn3r) [inline docs/config_nat.md:106]: **For** shared NAT pools provisioned on **an** HA interface that encounters a failover, the 128T software **will put** the interface **into** recovery mode to recover all sessions. At the end of the recovery period all non-discovered ports **(delete as)** designed as free are returned to the NAT pool.

File: docs/config_nat.md [hunk 3]
```diff
@@ -111,61 +94,28 @@ network-interface
...
 
 ### Tenant filtering
 The _nat-pool_ configuration can optionally be provisioned with a list of tenants. When the configuration has _multiple_ IP pools available, the _tenant_ can be used to determine which IP pool will be selected for the source NAT. Absence of a tenant implies that the IP pool is valid for all traffic. The following rules will be applied in order to determine the selection of the NAT pool:
 
 * The packet has a source tenant associated with it
   * If more than one IP pool has matching tenant
-    * Static nat-pool: Only the first IP pool is used and hash is applied to create a session. The remainder of the matches are discarded
-    * Dynamic nat-pool: Picks the first IP pool with available capacity.
+    * Only the first IP pool is used and hash is applied to create a session. The remainder of the matches are discarded
```
Comment (Chr1st0ph3rTurn3r) [inline docs/config_nat.md:113]: Combine this with the line above:
If more than one IP pool has a matching tenant, only the first IP pool is used and the hash is applied to create a session. The remainder of the matches are discarded.

File: docs/config_nat.md [hunk 4]
```diff
@@ -111,61 +94,28 @@ network-interface
...
-    * Dynamic nat-pool: Same as above
+    * Same as above
   * All IP pools have tenant configured but none of them match the source tenant
     * Log the failure, increment stat and drop the packet
 * The packet has no source tenant associated with it
   * At least one IP pool has no tenant configured to do the source NAT; otherwise this will result in the session being dropped.
 
-The tenant matching rules will apply to sub-tenants as well. For example, if an IP pool allows tenant engineering; traffic with source tenant lab.engineering will also match the pool.
\ No newline at end of file
+The tenant matching rules will apply to sub-tenants as well. For example, if an IP pool allows tenant engineering; traffic with source tenant lab.engineering will also match the pool.
```
Comment (Chr1st0ph3rTurn3r) [inline docs/config_nat.md:121]: ...if an IP pool allows tenant engineering**, then** traffic with source tenant...

**Review by Chr1st0ph3rTurn3r — CHANGES_REQUESTED** [review]
General Editorial note: For bulleted or numbered lists, they should either all have punctuation or all not have punctuation. So if one bullet has a sentence with a comma, then it should end with a period, and all other items in all the lists should have the appropriate punctuation (period, colon, etc.).

**Review by ptimmons — DISMISSED** [review]
Nothing beyond what Chris has already identified.

**Review by MichaelBaj — APPROVED** [review]

**Review by ptimmons — APPROVED** [review]

**Review by Chr1st0ph3rTurn3r — APPROVED** [review]

**Pull request merged**